application/utils.py

```diff
@@ -0,0 +1,17 @@
+import os
+import httpx
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+API_KEY = os.environ.get("API_KEY")
```

Comment:
Ensure that the `API_KEY` environment variable is set correctly. If `API_KEY` is `None`, the request will fail. Consider adding a check to handle the case where `API_KEY` is not set.